go/tasks/pluginmachinery/flytek8s/container_helper.go

```diff
@@ -220,6 +220,9 @@ func ToK8sContainer(ctx context.Context, taskContainer *core.Container, iFace *c
 	if err := AddCoPilotToContainer(ctx, config.GetK8sPluginConfig().CoPilot, container, iFace, taskContainer.DataConfig); err != nil {
 		return nil, err
 	}
+	if container.SecurityContext == nil && config.GetK8sPluginConfig().DefaultSecurityContext != nil {
```

Comment:
Should we do this for all containers that we find in PodTask plugin? not just the main container?

Reply:
I don't think so, as we allow this to be added for other containers? Seems wrong